custom/icds_reports/static/icds_reports/mobile/css/less.min.js

```diff
@@ -0,0 +1,18 @@
+/*!
```

Comment:
is this not already being downloaded by bower?

Reply:
I don't think so. It's not a dependency we use anywhere else.